frontend/src/metabase/lib/notifications.ts

```diff
@@ -265,21 +270,34 @@ export const formatNotificationCheckSchedule = ({
       break;
     }
     case "monthly": {
-      if (
-        schedule_hour != null &&
-        schedule_day != null &&
-        schedule_frame != null
-      ) {
+      if (schedule_hour != null && schedule_frame != null) {
         const ampm = formatTimeWithUnit(schedule_hour, "hour-of-day", options);
-        const day = formatDateTimeWithUnit(
-          schedule_day,
-          "day-of-week",
-          options,
-        );
+        const day = schedule_day
+          ? formatDateTimeWithUnit(schedule_day, "day-of-week", options)
+          : t`day`;
         const frame = formatFrame(schedule_frame);
         return t`Check monthly on the ${frame} ${day} at ${ampm}`;
       }
       break;
     }
   }
 };
+
+export const formatNotificationScheduleDescription = ({
+  schedule_type,
+  schedule_hour,
+}: ScheduleSettings) => {
+  switch (schedule_type) {
+    case "daily":
+    case "weekly":
+    case "monthly": {
+      if (schedule_hour != null) {
+        const ampm = formatTimeWithUnit(schedule_hour, "hour-of-day");
+        return t`at ${ampm}`;
```

Comment:
Can we add a context string here? The translators will see "at {0}" which may be confusing